buku.py

```diff
@@ -48,12 +49,27 @@
 colorize = True  # Allow color output by default
 
 # Default colour to print records
-ID_str = '\x1b[96;1m%d. \x1b[1;92m%s\x1b[0;2m [%s]\x1b[0m\n'
-ID_DB_str = '\x1b[96;1m%d. \x1b[1;92m%s\x1b[0m'
+ID_srch = '\x1b[36m%d.\x1b[0m'
+ID_str = ID_srch + '\x1b[1;92m%s\x1b[2m [%s]\x1b[0m\n'
```

Comment:
The DB ID is not showing in dim. In

2.GitHub - mewrev/dissection: The dissection of a simple "hello world" ELF binary. [305]

The `[305]` at the end is the DB id. It is shown in search results. You have to use `\x1b[0;2m`